tests/loki/iofq_test.py

```diff
@@ -81,13 +81,17 @@ def test_pipeline_can_compute_IofQ_in_event_mode(uncertainties, target, qxy: boo
     assert sc.allclose(
         sc.values(reference.data),
         sc.values(result.hist().data),
-        rtol=sc.scalar(1e-11),
-        atol=sc.scalar(1e-11),
+        # Could be 1e-11, but currently the workflow defaults to float32 data, as
+        # returned by ScippNexus.
+        rtol=sc.scalar(1e-7),
```

Comment:
Not sure I follow anymore. So the changes you made here have degraded the accuracy of the results?

Reply:
Yes, because I change `numerator / denominator` to `numerator /= denominator`. So previously we ended up converting to `float64` in that step.